src/feed/TickHistoryFormatter.js

```diff
@@ -1,16 +1,21 @@
 export class TickHistoryFormatter {
+    static getUTCDate = function (date) {
```

Comment:
shouldn't this be `static function getUTCDate(date)`?

Also notice the duplicate code below. I think this should be `static function getUTCDate(epoch)`, where you can simply do `TickHistoryFormatter.getUTCDate(tick.epoch)`

Reply:
@brucebinary done!